mirror-web-server/src/space/space.gateway.ts

```diff
@@ -27,6 +28,14 @@ export class SpaceGateway {
     private readonly logger: Logger
   ) {}
 
+  @SubscribeMessage(ZoneSpaceWsMessage.GET_ASSETS)
+  public getAssetPerSpaceWithRolesCheck(
```

Comment:
This doesn't match the code here: https://github.com/the-mirror-gdp/mirror-official/pull/167/files#diff-5b20352cc4fcf50ecbf7f4cfeed18e73d05383aa22c0d7c7a9b27a1a1490c24eR47-R52

Also, we should just commit to FOSS first and then pull in the changes into mirror-official, not have 2 separate PRs